larpix/configuration/configuration_v2b.py

```diff
@@ -0,0 +1,330 @@
+from bitarray import bitarray
+import os
+import errno
+import functools
+from collections import OrderedDict
+
+from .. import bitarrayhelper as bah
+from .. import configs
+from . import BaseConfiguration, Configuration_v2, _Smart_List
+from . import configuration_v2 as conf_v2
+
+
+class Configuration_v2b(BaseConfiguration):
```

Comment:
You might consider inheriting from the `Configuration_v2` class, instead. That way we don't have duplicates of the `all_data`, `some_data`, etc methods. It makes it more complicated if we find a bug in one of these things.